### PR TITLE
[Anaconda] - [aiohttp] -  Address Security Vulnerability - GHSA-5h86-8mv2-jq9f

### PR DESCRIPTION
**Pull request by gauravsaini04**
**Dev container name**:
 
 * Anaconda
 
 **Description**:
 
 This PR patches the following vulnerability:
 
 * [GHSA-5h86-8mv2-jq9f](https://github.com/advisories/GHSA-5h86-8mv2-jq9f) - related to the `aiohttp` package;
 
 This vulnerability comes from the continuumio/anaconda3 image used upstream for the anaconda devcontainer.
 
 _Changelog_:
 
 * Updated Dockerfile
   * Locked version for patched python package;
     * `aiohttp` - _minimum package version set to `3.9.2`_;
   * Removed locked version for previously patched conda package;
     * `aiohttp` - _minimum package version was set to `3.9.0`_;
 * Updated tests to verify `aiohttp` minimum version (Minimum package version set to `3.9.2` which fixes [GHSA-5h86-8mv2-jq9f](https://github.com/advisories/GHSA-5h86-8mv2-jq9f));
 
 **Checklist**:
 
 * [x]   Checked that applied changes work as expected

### CODE REVIEW

File: src/anaconda/test-project/test.sh [hunk 1]
```diff
@@ -43,7 +43,7 @@ checkPythonPackageVersion "requests" "2.31.0"
 checkPythonPackageVersion "cryptography" "41.0.7"
 checkPythonPackageVersion "transformers" "4.36.0"
```
Comment (samruddhikhandale) [inline src/anaconda/test-project/test.sh:44]: We have some failing checks, can you fix them? 

https://github.com/devcontainers/images/actions/runs/7813269949/job/21312054793?pr=949#step:3:4694

Looks like these have started failing on `main` as well. See https://github.com/devcontainers/images/actions/workflows/smoke-anaconda.yaml

Comment (samruddhikhandale) [reply src/anaconda/test-project/test.sh:44]: Can you update to this line https://github.com/devcontainers/images/blob/main/.github/workflows/smoke-anaconda.yaml#L17C1-L17C47 to 👇 

```
runs-on: devcontainer-image-builder-ubuntu
```

This should ideally fix `No space left on device')` error

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: src/anaconda/test-project/test.sh (hunk 1 above)
Comment (gauravsaini04) [reply src/anaconda/test-project/test.sh:44]: Have done. 👍

**Review by samruddhikhandale — APPROVED** [review]
⚡

**Pull request merged**